canvasapi/account.py

```diff
@@ -643,11 +643,11 @@ def create_user_login(self, user, login, **kwargs):
         :calls: `POST /api/v1/accounts/:account_id/logins \
         <https://canvas.instructure.com/doc/api/logins.html#method.pseudonyms.create>`_
 
-        :param user: The ID of the user to create the login for
+        :param id: The ID of the user to create the login for
```

Comment:
Sorry, I may not have been clear earlier.

`:param user:` is indeed correct, I meant that the description needs to be updated.